### PR TITLE
Update HAB_BLDR_URL env var

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

Clarifies setting the HAB_BLDR_URL environment variable correctly.

### CODE REVIEW

File: components/automate-chef-io/content/docs/on-prem-builder.md [hunk 1]
```diff
@@ -298,6 +298,16 @@ To finish up, return to your Chef Habitat Builder on-prem installation and view
...
 
+### Chef Habitat Environment Variables
+
+Set the Chef Habitat Builder on-prem `HAB_BLDR_URL` environment variable with:
+
+```
+HAB_BLDR_URL=https://{{< example_fqdn "automate" >}}/bldr/v1/
+```
+
+For more, see the Chef Habitat [environment variables](https://www.habitat.sh/docs/reference/) documentation.
```
Comment (susanev) [inline components/automate-chef-io/content/docs/on-prem-builder.md:309]:
```suggestion
For more, see the Chef Habitat [environment variables](https://www.habitat.sh/docs/reference/#environment-variables) documentation.
```

should we link to the specific section?

File: components/automate-chef-io/content/docs/on-prem-builder.md [hunk 2]
```diff
@@ -298,6 +298,14 @@ To finish up, return to your Chef Habitat Builder on-prem installation and view
 
 Chef Habitat Builder uses the same mechanisms that Chef Automate does for [backups]({{< relref "backup.md" >}}), [log management]({{< relref "log-management.md" >}}), and [uninstalling]({{< relref "troubleshooting.md#uninstalling-chef-automate" >}}).
 
+### Chef Habitat Environment Variables
+
+Set the Chef Habitat Builder on-prem `HAB_BLDR_URL` environment variable on the command line with:
```
Comment (yzl) [inline components/automate-chef-io/content/docs/on-prem-builder.md:303]: Might want to mention when someone would want to do this.

Comment (kagarmoe) [inline components/automate-chef-io/content/docs/on-prem-builder.md:303]:
```suggestion
If needed, set the Chef Habitat Builder on-prem `HAB_BLDR_URL` environment variable on the command line with:
```

Comment (kagarmoe) [reply components/automate-chef-io/content/docs/on-prem-builder.md:303]:
```suggestion
If needed for troubleshooting or communicating with your Chef Habitat Builder on-prem installation, set the `HAB_BLDR_URL` environment variable on the command line with:
```

**Pull request merged**